modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -132,6 +133,26 @@ spec:
         - 192.0.2.251
 # ...
 ----
++
+[IMPORTANT]
+====
+You can specify DNS options under the `dns-resolver.config` section of your NNCP file as demonstrated in the following example:
+
+[source,terminal]
+----
+# ...
+desiredState:
+    dns-resolver:
+      config:
+        search:
+         options:
+         - timeout:2
+         - attempts:3
+# ...
+----
+
+If you want to remove the options from your network interface, delete the options from the file, add `state: absent` under the `dns-resolver.config` section of the NNCP, and then run `oc apply -f <nncp_file_name>`. If you skip adding the `state: absent` step, the options persist on your network interface.
```

Comment:
DNS does not have "state: absent" as opposed to interfaces. For this example you can just link to the upstream nmstate documentation that has an example -- https://github.com/nmstate/nmstate/blob/base/examples/dns_remove.yml

Reply:
I added a configuration as linking to GH might not be warranted.